.github/workflows/main.yaml

```diff
@@ -0,0 +1,40 @@
+# Github action to test localrest
+#
+# Also upload the documentation to Github pages
+#
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  main:
+    name: Run
+    # Matrix configuration
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-18.04, macOS-10.15, windows-2019 ]
+        dc: [ ldc-master, dmd-master, ldc-latest, dmd-latest, ldc-1.17.0, dmd-2.088.1 ]
```

Comment:
How about gdc? We don't want to support it yet?

Reply:
s/we/you

Reply:
Riiiiiight, I'll add it back